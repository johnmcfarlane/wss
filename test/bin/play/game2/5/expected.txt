 40 10,1  | BEGAZE
 32 10,2  | GAZEE
 29  7,8  - GABE
 28  7,8  - BAG
 28  7,8  - BEG
 28  7,8  - GAB
 28  7,8  - GEB
 26  3,5  - BAZE
 26  5,5  - ZAG
 24  7,8  - BAE
 24  3,5  - GAZE
 23  3,5  - EZBA
 22  8,8  - EGBA
 22  7,8  - GAE
 22  5,5  - ZEA
 21  5,5  - GAB
 21  5,5  - GEB
 20  7,8  - BA
 20  5,5  - BAG
 20  7,8  - BE
 20  5,5  - BEG
 19  8,8  - AB
 19  7,5  | BALZA
 19  9,8  - GAZ
 19  9,8  - GEZ
 18  6,5  - AB
 18  8,8  - AGE
 18  3,5  - GABE
 18  4,5  - ZEA
 17  8,8  - AG
 17  5,5  - ZA
 16  7,5  - AB
 16  6,5  - AG
 16 10,3  | BAZE
 16  5,5  - BEA
 16  3,5  - EGBA
 15 10,3  | GAZE
 14  7,8  - AE
 14  7,5  - AG
 14  9,8  - BAG
 14  9,8  - BEG
 14  7,4  | ZEAL
 13  8,8  - AE
 13  8,3  | BEGUT
 13  9,8  - GAB
 13  8,4  | GAUT
 13  9,8  - GEB
 12  4,5  - BAE
 12  9,8  - BAE
 12  4,5  - BEA
 12  9,8  - BEA
 12  7,5  | GALBE
 12 10,4  | ZEE
 11  4,5  - AGE
 11  5,5  - BA
 11  9,8  - BA
 11  5,5  - BE
 11  9,8  - BE
 11  8,3  | BEAUT
 11  8,5  | BUTEA
 11  4,5  - GAE
 10  8,4  | ABUT
 10 10,5  - BA
 10 10,5  - BE
 10  8,5  | BUTE
 10  9,8  - GAE
 10  6,6  | PABA
  9  7,6  | ALBA
  9  7,6  | ALBE
  9  7,6  | ALGAE
  9  7,4  | BEALA
  9 10,3  | GABE
  9  7,5  | GALEA
  9  6,4  | GAPA
  9  6,6  | PAGA
  9  6,6  | PAGE
  8  5,5  - AE
  8  7,6  | ALGA
  8  7,4  | BAAL
  8  7,5  | BALA
  8  7,5  | BALE
  8  7,4  | BEAL
  7 10,5  - AB
  7  7,4  | AGAL
  7  7,6  | ALB
  7  8,5  | AUTE
  7  7,5  | BAL
  7  8,5  | BUT
  7  7,4  | EGAL
  7  7,5  | GALA
  7  7,5  | GALE
  7  7,4  | GEAL
  7  9,6  | SOGA
  6 10,5  - AG
  6 10,3  | AGEE
  6  7,6  | ALAE
  6  7,6  | ALEA
  6  7,6  | ALG
  6  6,5  | APA
  6 10,4  | BAE
  6 10,4  | BEE
  6  6,5  | EPA
  6  7,5  | GAL
  6  8,5  | GUT
  6  9,6  | SOB
  6  8,6  | UTA
  6  8,6  | UTE
  5 10,4  | AGE
  5 10,5  | BE
  5 10,4  | GAE
  5 10,4  | GEE
  5  9,6  | SOG
  4  7,5  | AAL
  4 10,5  - AE
  4  7,6  | ALA
  4  7,6  | ALE
  3  9,6  | SOE
  2 10,5  | AE
  2 10,5  | EE
