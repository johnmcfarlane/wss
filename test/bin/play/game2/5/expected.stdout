9,0:begaze=40
6,7:ba=20
6,7:be=20
7,7:ab=19
9,2:gaze=15
6,7:ae=14
6,4:ag=14
8,7:bag=14
8,7:beg=14
6,3:zeal=14
9,3:zee=12
7,2:beaut=11
7,3:abut=10
7,4:bute=10
6,5:alba=9
6,5:algae=9
6,4:galea=9
5,5:page=9
6,5:alga=8
6,3:baal=8
6,4:bale=8
8,5:soba=8
6,5:alb=7
6,4:bal=7
7,4:but=7
6,3:egal=7
6,4:gala=7
6,4:gale=7
9,2:agee=6
6,5:alae=6
9,3:bee=6
6,4:gal=6
7,4:gut=6
8,5:sob=6
7,5:uta=6
7,5:ute=6
9,3:age=5
9,3:gae=5
9,3:gee=5
6,4:aal=4
6,5:ala=4
6,5:ale=4
